.github/workflows/sql-release.yml

```diff
@@ -17,9 +17,9 @@ jobs:
           - os: linux
             arch: amd64
             runner: ubuntu-latest
-          - os: windows
-            arch: amd64
-            runner: windows-latest
+#          - os: windows todo fix windows builds, depends on setup-protoc action https://github.com/arduino/setup-protoc/issues/33
+#            arch: amd64
+#            runner: windows-latest
```

Comment:
Can you add Windows back now that we fixed the protoc version?

Reply:
added